tests/src/valueFormatters/valueFormatters.tests.js

```diff
@@ -137,15 +136,14 @@
 							+ 'DataValue.'
 					);
 
+					done();
 				} )
 				.fail( function( errorMessage ) {
```

Comment:
Looks like it could use `.always( done )` instead. If you place the callback (only) in `done()`, then the test should indeed fail still, but it would fail from a timeout, which might be confusing.

Reply:
Yea, this is much better. Thanks!